docs/docs/ecosystem/community.md

```diff
@@ -8,61 +8,111 @@ Sponsors, contributors, and adopters make this ecosystem possible and shape it
 in significant ways.
 
 Important community resources include [GitHub](https://github.com/Ory),
-[Slack (chat)](https://slack.Ory.sh/) and GitHub Discussions for
+[Ory Chat](https://slack.Ory.sh/) and GitHub Discussions for
```

Comment:
For most people, Slack is a clear indication of what to expect while "Ory Chat" is ominous. Maybe "Ory Slack Channel"?